pom.xml

```diff
@@ -985,6 +992,29 @@
                 <artifactId>maven-dependency-plugin</artifactId>
                 <version>${maven-dependency-plugin-version}</version>
             </plugin>
+
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <configuration>
+                    <java>
+                        <palantirJavaFormat>
+                            <version>2.30.0</version>
```

Comment:
Add to "properties" tag ?

Reply:
Make sense. As this PR already touch a large range of code. I tend to postpone this suggestion after we merge this one.